tools/destination-tester/README.md

```diff
@@ -153,3 +153,9 @@ This option disables encryption and compression of batch files for debugging pur
 
 #### --input-file
 The tester by default reads all input files from local data folder and executes them in the alphabetical order they appear. You can specify a single input file to be read and executed using this option. Providing just the filename is sufficient.
+
+#### --schema-name
+The tester by default creates `tester` named schema. This option allows the tester to run with different schema name.
+
+#### --disable-operation-delay
+The tester by default adds delay to operation for real time simulation, set this boolean to disable delay.
```

Comment:
@fivetran-abdulsalam 
'--disable-operation-delay' - what is the syntax for setting this key to 'true'?
Or just specifying this key disables the delay?

Reply:
[@alexander.ilyichov](https://fivetran.slack.com/team/U01AUGQFYHK)
 just specifying this key will disable it.
e.g.
docker run --mount type=bind,source=<local-data-folder>,target=/data -a STDIN -a STDOUT -a STDERR -it -e WORKING_DIR=<local-data-folder> -e GRPC_HOSTNAME=host.docker.internal --network=host fivetrandocker/sdk-destination-tester:<version> --disable-operation-delay